pkg/operator/configobservation/network/observe_network_test.go

```diff
@@ -61,17 +52,18 @@ func TestObserveRestrictedCIDRs(t *testing.T) {
 	}
 
 	result, errors = ObserveRestrictedCIDRs(listers, events.NewInMemoryRecorder("network"), map[string]interface{}{})
-
-	restrictedCIDRs, _, err := unstructured.NestedStringSlice(result, "admission", "pluginConfig", "network.openshift.io/RestrictedEndpointsAdmission", "configuration", "restrictedCIDRs")
-	if err != nil {
-		t.Fatal(err)
-	}
-	if restrictedCIDRs[0] != "podCIDR" {
-		t.Error(restrictedCIDRs[0])
-	}
-	if restrictedCIDRs[1] != "serviceCIDR" {
-		t.Error(restrictedCIDRs[1])
-	}
+	assert.Empty(t, errors)
+	shouldMatchYaml(t, result, `
```

Comment:
can you strictly unmarshal this into a real Golang config type everywhere? So we catch API type incompatibility and typos early.

Reply:
That would require either copy-pasting the type or vendoring in kubernetes, Which would you prefer?

Reply:
oh, it's just deep in k/k ? that wouldn't make it easy. And no, we don't want to vendor k/k :)